service/initialise.go

```diff
@@ -65,6 +69,21 @@ func (e *ExternalServiceList) GetKafkaConsumer(ctx context.Context, cfg *config.
 	return kafkaConsumer, nil
 }
 
+// GetS3Clients returns S3 clients public and private. They share the same AWS session.
+func (e *ExternalServiceList) GetS3Clients(cfg *config.Config) (s3Public S3Client, s3Private S3Client, err error) {
+	s3Public, err = e.Init.DoGetS3Client(cfg.AwsRegion, cfg.PublicBucketName, true, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+	e.S3Public = true
+	s3Private, err = e.Init.DoGetS3Client("", cfg.PrivateBucketName, true, s3Public.Session())
```

Comment:
No AWS region is passed in here?
```suggestion
	s3Private, err = e.Init.DoGetS3Client(cfg.AwsRegion, cfg.PrivateBucketName, true, s3Public.Session())
```

Reply:
It uses the session object, that has the region information already. Maybe the DoGetS3Client() should be split in DoGet.. and DoGetWithSession ?

Reply:
Updated with 2 separate functions: `DoGetS3Client` and `DoGetS3ClientWithSession`